### PR TITLE
Remove simple 9: "a simple" => "an"

### PR DESCRIPTION
**Pull request by estelle**

### CODE REVIEW

File: files/en-us/web/api/html_drag_and_drop_api/drag_operations/index.md [hunk 1]
```diff
@@ -74,7 +74,7 @@ The {{domxref("DataTransfer")}} contains two pieces of information, the **type**
...
 
-A drag may include data items of several different types. This allows data to be provided in more specific types, often custom types, yet still provide fallback data for drop targets that do not support more specific types. It is usually the case that the least specific type will be normal text data using the type [`text/plain`](/en-US/docs/Web/API/HTML_Drag_and_Drop_API/Recommended_drag_types#dragging_text). This data will be a simple textual representation.
+A drag may include data items of several different types. This allows data to be provided in more specific types, often custom types, yet still provide fallback data for drop targets that do not support more specific types. It is usually the case that the least specific type will be normal text data using the type [`text/plain`](/en-US/docs/Web/API/HTML_Drag_and_Drop_API/Recommended_drag_types#dragging_text). This data will be a textual representation.
```
Comment (wbamberg) [inline files/en-us/web/api/html_drag_and_drop_api/drag_operations/index.md:77]: I guess? I don't really know what "This data will be a textual representation" means. Maybe we could just delete that sentence?

**Review by wbamberg — COMMENTED** [review]
Thanks for this Estelle!

File: files/en-us/web/api/html_drag_and_drop_api/drag_operations/index.md (hunk 1 above)
Comment (estelle) [inline files/en-us/web/api/html_drag_and_drop_api/drag_operations/index.md:77]:
```suggestion
A drag may include data items of several different types. This allows data to be provided in more specific types, often custom types, yet still provide fallback data for drop targets that do not support more specific types. It is usually the case that the least specific type will be normal text data using the type [`text/plain`](/en-US/docs/Web/API/HTML_Drag_and_Drop_API/Recommended_drag_types#dragging_text).
```

Comment (estelle) [reply files/en-us/web/api/html_drag_and_drop_api/drag_operations/index.md]: Deleted. Thanks for the review!

**Review by wbamberg — APPROVED** [review]
👍 thanks!

**Comment by rebloor** [comment]
Perhaps "a single" next, only 1290 files affected.

**Pull request merged**